assetsbrowser.py

```diff
@@ -58,44 +59,19 @@ def __init__(self, parent=None):
 
         # 3. Project List Dropdown ComboBox
         self.project_path = configurations.get_setting('Settings', 'ProjectPath')
-        self.comboBox.fsm = QtWidgets.QFileSystemModel()
-        self.comboBox.rootindex = self.comboBox.fsm.setRootPath(self.project_path)
-        self.comboBox.setModel(self.comboBox.fsm)
-        self.comboBox.setRootModelIndex(self.comboBox.rootindex)
-        self.comboBox.setCurrentIndex(1)
-        self.comboBox.activated[str].connect(self.project_list)
-
-        # 4.1 Create empty list and dictionary for ColumnView tabs
-        self.category = []
-        self.assets = {}
-        self.current_project = self._current_project()
-        assets_path = os.path.join(self.project_path, self.current_project, "Assets")
-
-        # 4.2 Warn user if Assets directory doesn't exists
-        if not os.path.isdir(assets_path):
-            warning_text = (
-                    "Assets directory is unavailable.\n"
-                    + "\n"
-                    + "Please ensure you have access to it."
-            )
-            helpers.utils.alert_window("Warning", warning_text)
-            helpers.functions.close_app()
+        self.combobox_fsm = QtWidgets.QFileSystemModel()
+        self.comboBox.setModel(self.combobox_fsm)
+        root_idx = self.combobox_fsm.setRootPath(self.project_path)
+        self.comboBox.setRootModelIndex(root_idx)
+        self.combobox_fsm.directoryLoaded.connect(self.populate_project_list)
```

Comment:
`directoryLoaded` is crucial if need to do extra handling that must be done before the main window appear. Refer to `populate_project_list`